### PR TITLE
First cleanup

### PR DESCRIPTION
**Pull request by PabloPie**

### CODE REVIEW

**Comment by PabloPie** [comment]
Config files taken from deploy-alpha

File: roles/caliopen/templates/configs/apiv2.yaml [hunk 1]
```diff
@@ -5,41 +5,49 @@ ApiConfig:
...
+    admin_username: admin
+    templates_path: "/etc/caliopen/notifiers/templates/"
 ProxyConfig:
   host: 0.0.0.0
   port: 31415
   routes:
   - /api/v2/: 127.0.0.1:6544
-  - /: apiv1:6543         # route "/" catch all other requests
-
+  - /: {{ hostvars['api1']['backend_ip'] }}:6543         # route "/" catch all other requests
```
Comment (PabloPie) [inline roles/caliopen/templates/configs/apiv2.yaml]: Not sure how this one goes...

Comment (gdchamal) [reply roles/caliopen/templates/configs/apiv2.yaml]: we will not use the proxy of apiv2. We must figure out how to correctly disable it

File: deployments/frontend-deployment.yaml [hunk 2]
```diff
@@ -25,9 +34,9 @@ spec:
         - containerPort: 4000
         env:
         - name: CALIOPEN_API_HOSTNAME
-          value: "apiv2"
+          value: "api.beta.caliopen.org"
```
Comment (gdchamal) [inline deployments/frontend-deployment.yaml]: should use an ansible variable {{caliopen_domain_name}} not this hardcoded value

**Review by gdchamal — CHANGES_REQUESTED** [review]

File: deployments/frontend-deployment.yaml (hunk 2 above)
Comment (PabloPie) [reply deployments/frontend-deployment.yaml]: the problem is deployments are not in ansible scope, to do that we just need a new configmap passed to the deployment as an environment variable

**Pull request merged**